source/includes/dev/_fw_contextual_layers.md

```diff
@@ -0,0 +1,31 @@
+# Forest Watcher Contextual layers
+
+## Updating the Tree Cover Loss data
+
+The [FW Contextual layers microservice](https://github.com/gfw-api/fw-contextual-layers) serves tree cover loss (TCL) data in tile format through one of its endpoints. The underlying dataset that is used to create these tiles is updated regularly - typically once a year - through a process that requires minor modifications to the configuration of the service. Below we'll outline the steps to carry out this update
+
+### Update the Tree Cover Loss layer URL
+
+The first step is updating the source code of the Contextual layers microservice - specifically [this line](https://github.com/gfw-api/fw-contextual-layers/blob/a8442da2c7a4e68625867479860c0185f35488a8/config/default.json#L17) - and setting the updated URL for the updated dataset containing the tree cover loss data. This updated dataset will be used as a replacement of the previous one. On the more common scenario of the yearly updates to the TCL data, this means that the new dataset should have the data for all previously existing years, plus the newly added year.
```

Comment:
Change this URL to point to a branch, not a commit
```suggestion
The first step is updating the source code of the Contextual layers microservice - specifically [this line](https://github.com/gfw-api/fw-contextual-layers/blob/dev/config/default.json#L17) - and setting the updated URL for the updated dataset containing the tree cover loss data. This updated dataset will be used as a replacement of the previous one. On the more common scenario of the yearly updates to the TCL data, this means that the new dataset should have the data for all previously existing years, plus the newly added year.
```

Reply:
I would recommend we point to the commit, because it ensures we are pointing to the relevant line, always. Say we refactor that file on a future commit to dev, and the relevant line moves up/down -  now the link no longer points to the correct place. Move the relevant line enough, and we have ourselves a nice set of instructions on how to add a bug to the code.

Sure, pointing to the commit has the downside of potentially pointing to outdated code that no longer resembles the current one, but that would lead the dev to try and figure out where the logic was moved to.